.travis.yml

```diff
@@ -8,6 +8,10 @@ sudo: required
 services:
   - docker
 
+branches:
```

Comment:
I think this disables building of user branches and PR's - https://docs.travis-ci.com/user/customizing-the-build#building-specific-branches
What we need is building PR's and user branches but deploying only develop branch, so i think this section
```
  - PROJECT_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
  - oc login https://openshift.adorsys.de:443 --username=$OPENSHIFT_USER --password=$OPENSHIFT_PASSWORD
  - docker login -u $(oc whoami) -p $(oc whoami -t) https://openshift-registry.adorsys.de
  - docker build -t datasafe-rest-service:$TRAVIS_COMMIT --build-arg JAR_FILE=datasafe-rest-impl-$PROJECT_VERSION.jar ./datasafe-rest-impl
  - export IMAGE_NAME=openshift-registry.adorsys.de/datasafe/datasafe-rest-service
  - docker tag datasafe-rest-service:$TRAVIS_COMMIT $IMAGE_NAME:$TRAVIS_COMMIT
  - docker push $IMAGE_NAME:$TRAVIS_COMMIT
  - oc tag $IMAGE_NAME:$TRAVIS_COMMIT datasafe-rest-service:latest
```
should go somewhere under deploy section into deploy_develop_to_openshift.sh and deploy section should look like:
```
deploy:
  - provider: script
    skip_cleanup: true
    script: /bin/bash .travis/deploy.sh
    on:
      tags: true
      condition: "$TRAVIS_TAG =~ ^v([[:digit:]]+\\.)+[[:digit:]]+(-[[:digit:]]+)?(-SNAPSHOT)?$"
 - provider: script
    skip_cleanup: true
    script: /bin/bash .travis/deploy_develop_to_openshift.sh # in this script we play with OpenShift
    on:
      branch: develop
```

Reply:
updated the script to deploy develop alone.